tibo-mllr/activeviam-projet
# Fixing issue #40 : creating the summary page

# Issue Number:

Fix issue #40 

# Description:

Creating a summary page with query plan information

# How to test:

Launch the app
Go to localhost:3000/summary
Check that the page exists and is working

# Other notes:
